src/main/java/subway/lines/Line.java

```diff
@@ -0,0 +1,108 @@
+package subway.lines;
+
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import subway.section.Section;
+import subway.section.SectionAddRequest;
+import subway.section.SectionDeleteRequest;
+
+@Entity
+public class Line {
+    @Id
+    @GeneratedValue
+    private Long id;
+    private String name;
+
+    private String color;
+
+    private Long upStationId;
+    private Long downStationId;
+    private Long distance;
+
+    @OneToMany(mappedBy = "line")
```

Comment:
```suggestion
    @OneToMany(mappedBy = "line", cascade = {CascadeType.PERSIST, CascadeType.MERGE}, orphanRemoval = true)
```

이런형태로 해도 될 것 같아요 😄 
`구간은 노선에 포함되어있다` 라는 전제를 가져도 될만한 상황이기 때문에, 라이프 사이클을 공유하도록 구현하셔도 좋습니다.
위 옵션들에 대해서는 [참고](https://velog.io/@yuseogi0218/JPA-CascadeType.REMOVE-vs-orphanRemoval-true) 를 봐주세요 

이렇게 해두면 조금 더 객체지향적인 설계를 하실 수 있을 거에요
일단은 한번 도전해보시고, 너무 오래 걸리거나 힘드시면 따로 말씀해주세요 😅 
참고로 말씀드리면 목요일 강의전에 해당 주차의 미션을 무조건 다 완수하지 않으셔도 됩니다.